driver/src/main/java/org/neo4j/driver/v1/Driver.java

```diff
@@ -72,6 +73,11 @@
 public interface Driver extends AutoCloseable
 {
     /**
+     * Return a flag to indicate whether or not encryption is used for this driver.
+     */
+    boolean encrypted();
```

Comment:
I think this method should be named `#isEncrypted()` to signify that it is a configuration option check and not some soft of a fluent interface.